src/3rdparty/walkontable/src/border.js

```diff
@@ -580,6 +581,30 @@ class Border {
     return false;
   }
 
+  /**
+   * Change border style.
+   *
+   * @private
+   * @param {HTMLElement} borderElement
+   */
+  changeBorderStyle(borderElement, border) {
+    let style = this[borderElement].style;
```

Comment:
`borderElement` has improper jsdoc tag. It seems that this argument holds only string values?

Reply:
Yes, actually. It's just coordinates where add / remove borders.